src/patternfly/components/Dropdown/dropdown-toggle-button.hbs

```diff
@@ -1,16 +1,20 @@
-<button class="pf-c-dropdown__toggle-button{{#if dropdown-toggle-button--modifier}} {{dropdown-toggle-button--modifier}}{{/if}}"
-  id="{{dropdown-toggle-button--id}}"
-  {{#if aria-label}}
-    aria-label="{{aria-label}}"
-  {{/if}}
-  {{#if dropdown--IsToggleButton}}
+<button class="pf-c-dropdown__toggle-button{{#if dropdown-toggle-button--IsAction}} pf-m-action{{/if}}{{#if dropdown-toggle-button--modifier}} {{dropdown-toggle-button--modifier}}{{/if}}"
+  {{#unless dropdown-toggle-button--IsAction}}
+    id="{{id}}-toggle-button"
```

Comment:
Should this `id="{{id}}-toggle-button"` moved out of `{{#unless ...}}` conditional?

Reply:
I don't think do since the ID is only needed when the button toggles a menu. Ended up reverting this conditional back to support `dropdown--IsToggleButton` since that's already used in a bunch of places and put the `id` in that conditional.